.github/workflows/e2e_core.yml

```diff
@@ -176,6 +174,8 @@ jobs:
 
     - name: Run e2e tests
       id: tests
+      # Allow failures, since SYCL tests and API may be not stable
+      continue-on-error: true
```

Comment:
any other step should also allow failures...? `Build SYCL` perhaps...?

Reply:
I think these would make sense.

* Build SYCL
* Run check-sycl
* Build e2e tests

Reply:
Done